modules/SetupWizard/Android.mk

```diff
@@ -2,7 +2,12 @@ LOCAL_PATH := .
 include $(CLEAR_VARS)
 include $(GAPPS_CLEAR_VARS)
 LOCAL_MODULE := SetupWizard
-LOCAL_PACKAGE_NAME := com.google.android.setupwizard
+ifneq ($(filter $(PRODUCT_CHARACTERISTICS),tablet),)
+	LOCAL_PACKAGE_NAME := com.google.android.setupwizard.tablet
+else
+	LOCAL_PACKAGE_NAME := com.google.android.setupwizard.phone
```

Comment:
Just checking that you were awake ;) (Let's at least go with that explanation...). Anyways, fixed.